pkg/api/types.go

```diff
@@ -504,20 +506,32 @@ func (l *VolumeList) Set(value string) error {
 	if len(value) == 0 {
 		return errors.New("invalid format, must be source:destination")
 	}
+
+	var pos int
+	if isWindowsRegExp.MatchString(value) {
+		// We need to skip the 'C:\' that prefixes Windows directories, so we only
+		// check for ':' after that prefix.
+		pos = strings.Index(value[3:], ":") + 3
+	} else {
+		pos = strings.Index(value, ":")
```

Comment:
My point was that if it was `LastIndex` it must have been for a reason.
IIRC we are now accepting file names that contain `:` (since we added support for Windows).

In Linux and macOS file systems is it totally fine to have `:` in file names:

```
$ touch :hello
$ ls
:hello
```

So in this case, we could have input like `/src-:hello:/dest:z` -- I don't see a test case for that in `pkg/api/types_test.go`.